app/assets/javascripts/users/settings/account/preferences/index.js

```diff
@@ -71,7 +71,8 @@
   // Setup notification checkbox buttons
   function notificationsSettings() {
     var notification_settings = [ "recent_notification",
-                                  "assignments_notification" ]
+                                  "assignments_notification",
+                                  "popover_v2_toggle" ]
```

Comment:
Just from code structure point of view, I'd move this out of `notificationSettings()`, but into its own function. And I'd rename `popover_v2_toggle` to `popovers_enabled`.

Reply:
Hmm, the way im thinking about implementing its own settings function, it would require code duplication in index.js, the function for popovers_enabled would do the excact thing as the notifications function, but it would have a few different names, the new function would still read from the same place in prefferences controller as the old one does. Its around 50 lines of code duplicated

Reply:
I can do it, but it would be alot more code, maybe i can add comments to old used function?